src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/NetworkInterfacePal.Linux.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Net.NetworkInformation
+{
+    internal static class NetworkInterfacePal
+    {
+        /// Returns objects that describe the network interfaces on the local computer.
```

Comment:
Based on other changes in this PR and new guidance we should either use `//` (not `///`) or we should properly add the XML documentation tags.